features/step_definitions/pull_request_steps.rb

```diff
@@ -0,0 +1,3 @@
+Then(/^I see a new (.+?) pull request for the "(.+?)" branch in the "(.+?)" repo in my browser$/) do |domain, branch, repo|
+  expect(@last_run_result.out).to eql "#{@tool} called with: #{pull_request_url domain, branch, repo}\n"
+end
```

Comment:
I disagree with 
```
Everything in the specs is about something that comes from running a git town command
```
The specs verify two things. The state after the git town command runs and the output of the command. A lot of the specs are about verifying the state. The output of the command is limited to: what get printed to the terminal, opening a browser and opening a manpage.

`run_steps.rb` is not a very long file and I said in my initial comment at the moment is contains all references to @last_run_result which is used for verifying the output of the command. I am returning to my original suggestion and would like this moved back to `run_steps.rb`